src/data/geo/geometry.js

```diff
@@ -0,0 +1,38 @@
+import SpatialReferenceSystem from './spatial-reference-system'
+import GeoJSONParser from './geo-json-parser'
+import WKTParser from './wkt-parser'
+
+class Geometry {
+  static fromWKT(wellKnownText, srs) {
+    return new WKTParser(srs).read(wellKnownText)
+  }
+
+  static fromGeoJSON(geoJSON, srs) {
+    return new GeoJSONParser(srs).read(geoJSON)
+  }
+
+  constructor(srs) {
+    this.srs = srs || SpatialReferenceSystem.DEFAULT
+  }
+
+  getSRS() {
+    return this.srs
+  }
+
+  toGeoJSON() {
+    return JSON.stringify({
+      type       : this.getGeojsonType(),
+      coordinates: this.jsonCoordinatePairs()
+    })
+  }
+
+  toWKT() {
+    return this.getWktType() + '(' + this.wktCoordinatePairs() + ')'
```

Comment:
any abstract methods like `wktCoordinatePairs|jsonCoordinatePairs|...` must be specified in the class, and they might be marked with JSDoc `@abstract` and also there should be a comment or exception `It is an abstract method and it must be overridden in an inherited class`, it will be more clear what methods this has.

Reply:
Agree with you, done.